Error runing dashboard workflow

Error:
remote: Permission to UCT-datastewardship/UCT-datastewardship.github.io.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/UCT-datastewardship/UCT-datastewardship.github.io/': The requested URL returned error: 403
Error: Process completed with exit code 128.

--- a/.github/workflows/update_dashboard.yml
+++ b/.github/workflows/update_dashboard.yml
@@ -26,7 +26,7 @@ jobs:
 
       - name: Commit and push if changed
         env:
-          GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           git config --global user.name "GitHub Actions"
           git config --global user.email "actions@github.com"

